### ISSUE TITLE
Add option to configure the MiniMap aria label

### ISSUE THREAD

**Issue by chrtze** [wbkd/react-flow#2544]
The current version of the MiniMap component has a default aria-label "React Flow mini map" that cannot be changed. Let's add an option to overwrite this with a custom string.
